gcc/rust/typecheck/rust-hir-path-probe.h

```diff
@@ -154,35 +146,33 @@ class PathProbeType : public TypeCheckBase, public HIR::HIRImplVisitor
   DefId specific_trait_id;
 };
 
-class ReportMultipleCandidateError : private TypeCheckBase
-{
+class ReportMultipleCandidateError : private TypeCheckBase {
 public:
-  static void Report (std::set<PathProbeCandidate> &candidates,
-		      const HIR::PathIdentSegment &query, Location query_locus)
-  {
-    RichLocation r (query_locus);
+  static void Report(std::set<PathProbeCandidate> &candidates,
+                     const HIR::PathIdentSegment &query, Location query_locus) {
+    RichLocation r(query_locus);
     for (auto &c : candidates)
-      r.add_range (c.locus);
+      r.add_range(c.locus);
 
-    rust_error_at (r, "multiple applicable items in scope for: %s",
-		   query.as_string ().c_str ());
+    rust_error_at(r, ErrorCode("E0034"),
```

Comment:
Only changed this code. Others changes are the result of running [clang-format10](https://github.com/DoozyX/clang-format-lint-action/blob/master/clang-format/clang-format10)